spec/abstractops.html

```diff
@@ -192,7 +176,7 @@ <h1>ToShowOffsetOption ( _normalizedOptions_ )</h1>
       It is different from the `offset` property passed to `Temporal.ZonedDateTime.from()`.
     </emu-note>
     <emu-alg>
-      1. Return ? GetOption(_normalizedOptions_, *"offset"*, « String », « *"auto"*, *"never"* », *"auto"*).
+      1. Return ? GetOption(_normalizedOptions_, *"offset"*, *"string"*, « *"auto"*, *"never"* », *"auto"*).
```

Comment:
why are these strings and not spec enums?

Reply:
Because that is the state of ECMA-402 until https://github.com/tc39/ecma402/pull/689 lands.